yt_dlp/extractor/xumo.py

```diff
@@ -0,0 +1,155 @@
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+    url_or_none,
+    determine_ext
+)
+
+
+class XumoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?xumo\.tv/(?P<mediaType>[^/]*)/[^/]*/(?P<id>[A-Z0-9]+)'
+    _TESTS = [{
+        # movie
+        'url': 'https://www.xumo.tv/free-movies/zombie-high/XM01OEG1MIJVJR',
+        'md5': '2843529c1abf5f86f61165637f89cbae',
+        'info_dict': {
+            'id': 'XM01OEG1MIJVJR',
+            'title': 'Zombie High',
+            'description': 'An academy coed (Virginia Madsen) discovers a faculty plot to achieve immortality with a serum made from students\' brains.',
+            'duration': 5436,
+            'release_year': 1987,
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'ext': 'mp4'
+        }
+    }, {
+        # series
+        'url': 'https://www.xumo.tv/tv-shows/super-mario-world/XM0AN69OG47PRN',
+        'playlist_count': 10,
+        'info_dict': {
+            'id': 'XM0AN69OG47PRN',
+            'title': 'Super Mario World'
+        },
+        'params': {
+            'skip_download': True
+        }
+    }]
+
+    _INFO_URL = 'https://valencia-app-mds.xumo.com/v2/assets/asset/'
+    _INFO_QUERY_PARAMS = {
+        'f': ['connectorId',
+              'title',
+              'providers',
+              'descriptions',
+              'runtime',
+              'originalReleaseYear',
+              'cuePoints',
+              'ratings',
+              'hasCaptions',
+              'availableSince',
+              'genres',
+              'season:all',
+              'episodes.episodeTitle',
+              'episodes.runtime',
+              'episodes.descriptions',
+              'episodes.hasCaptions',
+              'episodes.ratings']
+    }
+
+    def _get_video_info(self, video_id, info_json):
+        formats, subtitles = [], {}
+        has_drm = False
```

Comment:
unused

Reply:
Which part is unused?

Reply:
```py
        has_drm = False
```

Reply:
Whoops, looks like I didn't finish implementing that. My bad. Just pushed the missing logic.